code/modules/mob/living/living.dm

```diff
@@ -774,3 +774,6 @@ default behaviour is:
 		return
 	..()
 
+/mob/living/Life()
```

Comment:
and this tbh. the less stuff in bay files the better. I'm working through the merge conflicts on another branch and would prefer adding as little as possible.